media-sound/freac/freac-1.1~alpha20180913.recipe

```diff
@@ -0,0 +1,94 @@
+SUMMARY="Free and open source audio converter"
+DESCRIPTION="fre:ac is a free and open source audio converter. It comes with \
+an integrated tag editor and converts between various formats.
+
+Features include:
+
+- Support for MP3, MP4/AAC, Opus, FLAC and other formats
+- Integrated player and tag editor
+- Support for playlists, cue sheets and chapters
+- Fast, multi-threaded conversion engine
+- Ability to convert to multiple formats at once
+- Can verify lossless input and output files
+- Support for freedb/CDDB queries and submits
+- Support for using command line codecs
+- User interface available in 40+ languages
+- Optional command line interface"
+HOMEPAGE="https://freac.org/"
+COPYRIGHT="2001-2018 by Robert Kausch"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://github.com/enzo1982/freac/releases/download/v1.1-alpha-20180913/freac-1.1-alpha-20180913.tar.gz"
+CHECKSUM_SHA256="5ca1f3b8f548815ced94e8df3b0b736d75879816939661ba1f1baa94fd36f168"
+SOURCE_DIR="freac-1.1-alpha-20180913"
```

Comment:
Hmm, I forgot this one.
```diff
-SOURCE_URI="https://github.com/enzo1982/freac/releases/download/v1.1-alpha-20180913/freac-1.1-alpha-20180913.tar.gz"
+SOURCE_URI="https://github.com/enzo1982/freac/releases/download/v${portVersion/\~alpha/-alpha-}/freac-${portVersion/\~alpha/-alpha-}.tar.gz"
CHECKSUM_SHA256="5ca1f3b8f548815ced94e8df3b0b736d75879816939661ba1f1baa94fd36f168"
-SOURCE_DIR="freac-1.1-alpha-20180913"
+SOURCE_DIR="freac-${portVersion/\~alpha/-alpha-}"
```
This can be handy because it works in all cases, even if `$portVersion` does not contain `~alpha`.